Source/Core/Core/Config/UISettings.cpp

```diff
@@ -15,6 +15,77 @@ const Info<bool> MAIN_RECURSIVE_ISO_PATHS{{System::Main, "General", "RecursiveIS
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_2{
+    {System::Main, "Android", "ButtonToggleClassicButtonX"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_3{
+    {System::Main, "Android", "ButtonToggleClassicButtonY"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_4{
+    {System::Main, "Android", "ButtonToggleClassicButtonPlus"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_5{
+    {System::Main, "Android", "ButtonToggleClassicButtonMinus"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_6{
+    {System::Main, "Android", "ButtonToggleClassicButtonHome"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_7{
+    {System::Main, "Android", "ButtonToggleClassicTriggerL"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_8{
+    {System::Main, "Android", "ButtonToggleClassicTriggerR"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_9{
+    {System::Main, "Android", "ButtonToggleClassicButtonZL"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_10{
+    {System::Main, "Android", "ButtonToggleClassicButtonZR"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_11{{System::Main, "Android", "ButtonToggleClassicDPad"},
+                                               true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_12{
+    {System::Main, "Android", "ButtonToggleClassicStickLeft"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_CLASSIC_13{
+    {System::Main, "Android", "ButtonToggleClassicStickRight"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_0{{System::Main, "Android", "ButtonToggleWiimoteButtonA"},
+                                          true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_1{{System::Main, "Android", "ButtonToggleWiimoteButtonB"},
+                                          true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_2{{System::Main, "Android", "ButtonToggleWiimoteButton1"},
+                                          true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_3{{System::Main, "Android", "ButtonToggleWiimoteButton2"},
+                                          true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_4{
+    {System::Main, "Android", "ButtonToggleWiimoteButtonPlus"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_5{
+    {System::Main, "Android", "ButtonToggleWiimoteButtonMinus"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_6{
+    {System::Main, "Android", "ButtonToggleWiimoteButtonHome"}, true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_7{{System::Main, "Android", "ButtonToggleWiimoteDPad"},
+                                          true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_8{{System::Main, "Android", "ButtonToggleNunchukButtonC"},
+                                          true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_9{{System::Main, "Android", "ButtonToggleNunchukButtonZ"},
+                                          true};
+const Info<bool> MAIN_BUTTON_TOGGLE_WII_10{
+    {System::Main, "Android", "ButtonToggleWiimoteNunchuckStick"}, true};
```

Comment:
Okay, maybe the idea I had in the past of adding every Android setting to the C++ settings list was a bad one :) There's a lot of repetition here, and we don't actually use all these objects for anything... I would appreciate the input from someone who doesn't deal much with the Android side of things here.

Reply:
Oh, I'm in favor of removing these Android-only settings if there's no plans to use them in the future from the C++ side. Regex can convert the settings between C++ and Java if needed. Should I just remove them?

Reply:
Sounds fine to me. We can always re-add them if we need them.